tensorflow/compiler/xla/service/llvm_ir/llvm_util.h

```diff
@@ -312,6 +312,10 @@ llvm::GlobalVariable* GetOrCreateVariableForPhiloxRngState(
 // should rarely produce the same result.
 void IncrementVariableForPhiloxRngState(int64 value, llvm::Module* module,
                                         llvm::IRBuilder<>* b);
+
+// Obtain the target specific address space for LLVM IR generation of  global
```

Comment:
Nit: In Google style, comments on function prototypes like this one must be written in indicative mood, so "Obtains".